keps/sig-api-machinery/3962-mutating-admission-policies/README.md

```diff
@@ -496,7 +527,8 @@ List with "map" merge type:
   - For associatedList with multiple keys like example above, a directive field added could be used to indicate the deletion.
 ```yaml
 mutations:
-  - mutaton: >
+  - patchType: "ApplyConfiguration"
+    mutation: >
```

Comment:
same comment for all instances using mutation instead of expression for this path